articles/java/sdk/troubleshooting-messaging-event-hubs-processor.md

```diff
@@ -21,6 +21,17 @@ This article provides solutions to common problems that you might encounter when
 
 When the number of `EventProcessorClient` instances changes (that is, are added or removed), the running instances try to load-balance partitions between themselves. For a few minutes after the number of processors changes, partitions are expected to change owners. After it's balanced, partition ownership should be stable and change infrequently. If partition ownership is changing frequently when the number of processors is constant, it likely indicates a problem. We recommended that you file a GitHub issue with logs and a repro.
 
+Partition ownership is determined via the ownership records in the `CheckpointStore`.  Every load balancing interval it will:
+
+1. Fetch the latest ownership records.
+1. Check the records to see which ones have not updated its timestamp within the partition ownership expiration interval.  Records matching this criteria are considered.
+   1. If there are any unowned partitions and the load is not balanced between instances of `EventProcessorClient`, it will try to claim a partition.
+2. Update the ownership record for the partitions it owns that have an active link to that partition.
```

Comment:
```suggestion
1. Fetch the latest ownership records.
2. Check the records to see which ones have not updated its timestamp within the partition ownership expiration interval.  Records matching this criteria are considered.
   1. If there are any unowned partitions and the load is not balanced between instances of `EventProcessorClient`, it will try to claim a partition.
3. Update the ownership record for the partitions it owns that have an active link to that partition.
```